### PR TITLE
chore(demo): add analytics

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

## What is the new behavior?
yandex metrics added

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/app/app.component.ts [hunk 1]
```diff
@@ -44,4 +59,18 @@ export class AppComponent {
             (today.getMonth() === 11 && today.getDate() > 24)
         );
     }
+
+    private initRouterSubscribtion() {
+        this.router.events
+            .pipe(
+                filter(event => event instanceof NavigationEnd),
+                map(event => event as NavigationEnd),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/app.component.ts:67]:
```ts
filter((event): event is NavigationEnd => event instanceof NavigationEnd)
```

File: projects/demo/src/modules/app/app.browser.module.ts [hunk 2]
```diff
@@ -50,6 +51,7 @@ import {VersionManagerModule} from './version-manager/version-manager.module';
         VersionManagerModule,
         CustomHostModule,
         HttpClientModule,
+        MetrikaModule.forRoot({id: 87890624, webvisor: false, clickmap: false}),
```
Comment (nsbarsukov) [inline projects/demo/src/modules/app/app.browser.module.ts:54]: should we save it as environment variable ?

File: projects/demo/src/modules/app/app.component.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/app/app.component.ts]: thanks

File: projects/demo/src/modules/app/app.browser.module.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/app/app.browser.module.ts]: id moved to env

File: projects/demo/src/modules/app/app.component.ts [hunk 3]
```diff
@@ -44,4 +59,17 @@ export class AppComponent {
...
         );
     }
+
+    private initRouterSubscribtion() {
+        this.router.events
+            .pipe(
+                filter((event): event is NavigationEnd => event instanceof NavigationEnd),
+                takeUntil(this.destroy$),
+            )
+            .subscribe(event => {
```
Comment (waterplea) [inline projects/demo/src/modules/app/app.component.ts:69]: Distruct :)

**Review by waterplea — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**